scripts/tests/refresh-flux-ghcr-auth/wrapper_test.go

```
@@ -0,0 +1,471 @@
+	}
+	if !info.Mode().IsRegular() {
+		t.Fatalf("production KSail wrapper %q is not a regular file", ksailPullWrapperPath)
+	}
+
+	commands := [][]string{
+		{"cluster", "create"},
+		{"workload", "reconcile"},
+		{"cluster", "update"},
+	}
+	for _, command := range commands {
+		command := command
```

golangci-lint (2.12.2)

[error] The copy of the 'for' variable "command" can be deleted (Go 1.22+)

(copyloopvar)